### DISCUSSION TITLE
What coordinate system does IfcOBjectPlacement absolute placement uses?

### DISCUSSION THREAD

**Discussion by tomerAlef** [IfcOpenShell/IfcOpenShell#2817] [General]
According to the docs the object placement can be given an absolute placement which is relative to the world coordinate system but the coordinate system itslef isn't defined.

Where can I find/defind the coordinate system in use(UTM, ECEF, cartographic)?

**Comment by Moult** [comment]
The coordinate system is specified using IfcProjectedCRS in IFC4. Optionally, the coordinate system may also be a local coordinate system using IfcMapConversion which specifies a helmert transformation to/from the IfcProjectedCRS.
